projects/packages/publicize/src/class-publicize-setup.php

```diff
@@ -183,9 +183,63 @@ public static function add_filters_and_actions_for_screen( $current_screen ) {
 			return;
 		}
 
-		if ( Current_Plan::supports( 'republicize' ) ) {
-			add_filter( 'jetpack_post_list_display_share_action', '__return_true' );
+		/**
+		 * Filter to enable/disable the Share action on the post list screen.
+		 *
+		 * The Share action allows users to reshare published posts via Jetpack Social.
+		 * It is automatically enabled for plans that support the 'republicize' feature,
+		 * but can also be enabled/disabled via this filter.
+		 *
+		 * @since 0.2.0 Originally in jetpack-post-list package.
+		 * @since $$NEXT_VERSION$$ Moved to jetpack-publicize package.
+		 *
+		 * @param bool   $show_share Whether to show the share action. Default false.
+		 * @param string $post_type  The current post type.
+		 */
+		$show_share_action = Current_Plan::supports( 'republicize' )
+			|| apply_filters( 'jetpack_post_list_display_share_action', false, $current_screen->post_type );
```

Comment:
Previously, if I as a developer wanted to remove that "Share" action, I could do this

```php
add_filter( 'jetpack_post_list_display_share_action', '__return_false', 20 );
```

Now it doesn't work because of the OR condition. IMHO, it should be


```suggestion
		$show_share_action = Current_Plan::supports( 'republicize' )
			&& apply_filters( 'jetpack_post_list_display_share_action', true, $current_screen->post_type );
```

Reply:
Yeah good catch. Let's do that to retain that functionality, maybe someone uses it

Reply:
Fixed in c3cdab7